tests/packaging/package_test.go

```diff
@@ -44,7 +45,7 @@ func TestDeb(t *testing.T) {
 }
 
 func TestRpm(t *testing.T) {
-	rpms := getFiles(t, regexp.MustCompile(`\.rpm$`))
+	rpms := getFiles(t, regexp.MustCompile(`(i386|i686|amd64|x86)+.*\.rpm$`))
```

Comment:
```suggestion
        // only test x86 packages, excluding arm
	rpms := getFiles(t, regexp.MustCompile(`(i386|i686|amd64|x86)+.*\.rpm$`))
```

Reply:
Let's revert this as the jenkins build doesn't need it any more